nimbus-ui/nimbusui/src/app/components/platform/form/elements/picklist.component.ts

```diff
@@ -46,36 +48,42 @@ export const CUSTOM_INPUT_CONTROL_VALUE_ACCESSOR: any = {
-        <label [attr.for]="element.config?.code"  [ngClass]="{'required': requiredCss, '': !requiredCss}">{{label}}
+        <label [attr.for]="parent.config?.code"  [ngClass]="{'required': requiredCss, '': !requiredCss}">{{label}}
             <nm-tooltip *ngIf="helpText" [helpText]='helpText'></nm-tooltip>
         </label>
-        <div>
-            <fieldset [disabled]="!element?.enabled">
-                <p-pickList #picklist [source]="element.values" 
-                    filterBy="label"
-                    [sourceHeader] = "element.config?.uiStyles?.attributes.sourceHeader" 
-                    [targetHeader]="element.config?.uiStyles?.attributes.targetHeader" 
-                    [disabled]="!element?.enabled"
-                    [target]="targetList" pDroppable="dd" [responsive]="true" 
-                    [showSourceControls]="element.config?.uiStyles?.attributes?.showSourceControls"
-                    [showTargetControls]="element.config?.uiStyles?.attributes?.showTargetControls"
-                    (onMoveToTarget)="updateListValues($event)" (onMoveToSource)="updateListValues($event)">
-                    <ng-template let-itm pTemplate="item">
-                        <div class="ui-helper-clearfix">
-                            <div style="font-size:14px;float:right;margin:15px 5px 0 0" pDraggable="dd"  
-                                (onDragStart)="dragStart($event, itm)" (onDragEnd)="dragEnd($event)">{{itm.label}} </div>
+        <div> 
+        <fieldset [disabled]="!parent?.enabled">
+            <p-pickList #picklist 
+                [source]="parent.values" 
+                filterBy="label"
+                [sourceHeader] = "parent?.config?.uiStyles?.attributes.sourceHeader" 
+                [targetHeader]="parent?.config?.uiStyles?.attributes.targetHeader" 
+                [disabled]="!parent.enabled"
+                [target]="targetList" 
+                pDroppable="dd" 
+                [responsive]="true" 
+                [showSourceControls]="false"
+                [showTargetControls]="false"
+                (onMoveToTarget)="updateListValues($event)" 
+                (onMoveToSource)="updateListValues($event)">
+                <ng-template let-itm pTemplate="item">
+                    <div class="ui-helper-clearfix">
+                        <div pDraggable="dd"  
+                            (onDragStart)="dragStart($event, itm)" (onDragEnd)="dragEnd($event)">
+                            {{itm.label ? itm.label : getDesc(itm)}}
                         </div>
-                    </ng-template>
-                </p-pickList> 
-             </fieldset>   
+                    </div>
+                </ng-template>
+            </p-pickList>   
+            </fieldset>   
         </div>
    `
 })
 
 export class OrderablePickList extends BaseElement implements OnInit, ControlValueAccessor {
-
     @Input() element: Param;
```

Comment:
The field `element` is already available in `BaseElement`. Consider removing `element` from `OrderablePickList`.